app/collections/manager.cjsx

```diff
@@ -18,10 +18,10 @@ module?.exports = React.createClass
     project: React.PropTypes.object
 
   addToCollections: ->
-    options = @refs.search.options()
-    return unless options
+    collections = @refs.search.getSelected()
```

Comment:
This feels weird to me. Components shouldn't reach directly into each other's state, as that introduces coupling that could get messy later on.

Since we need to know what collections are selected in the `CollectionsManager`, maybe that array should be state here, and not in `CollectionSearch`, having it passed up via callback? Or better yet, maybe just move the submit button down into `CollectionSearch`, since that state isn't used anywhere else other than to fire the request off.